meilisearch-release-changelog/v1.2.0.md

```diff
@@ -122,8 +122,8 @@ Done by @loiclec, @dureuill, @ManyTheFish, @Kerollmops and @irevoire in #3542.
 - The `exactness` ranking rule no longer treats synonyms as exact matches. This boosts documents containing the query exactly as typed by the user. Solves https://github.com/meilisearch/meilisearch/issues/3410
 - When using the `attributes`, `exactness`, `typo`, and `proximity` ranking rules, Meilisearch will sort results as if  the`words` ranking rule were present and with higher priority. Search behavior when using the specified ranking rules before or without `words` was confusing and hurt relevancy
```

Comment:
Could we add something like "even if `words` ranking rule has been removed..."
what do you think @guimachiavelli and @dureuill?

Reply:
Attempting:

```
Meilisearch will always sort result as if the `words` ranking rule were present with a higher priority than the `attributes`, `exactness`, `typo` and `proximity` ranking rules, even if in the settings the `words` ranking rule has been removed or set with a lower priority. Search behavior when using the specified ranking rules before or without `words` was confusing and hurt relevancy
```

Reply:
For me it's perfect 👌